keyword_spotting_data_generator/evaluation/url_file_generator.py

```diff
@@ -56,14 +56,14 @@ def main():
             continue
 
         try:
-            srt_captions = caption.generate_srt_captions().split('\n\n')
+            srt_captions = caption.generate_srt_captions().lower().split('\n\n')
         except Exception as exception:
             cp.print_error("failed to retrieve for vidoe - ", url)
             continue
 
         keyword_exist = False
         for captions in srt_captions:
-            if keyword not in captions and keyword + "s" not in captions and keyword + "es" not in captions:
+            if keyword in captions or keyword + "s" in captions or keyword + "es" in captions:
```

Comment:
Maybe you could use something like `inflect` to generate more accurate plural forms of words?

https://pypi.org/project/inflect/